content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,346 @@
+
+* `active_time_seconds`
+* `compute_time_seconds`
+* `written_data_bytes`
+* `data_transfer_bytes` 
+
+These consumption metrics represent total cumulated usage across all branches and computes in a given project, accrued so far in a given monthly billing period. They are refreshed on a set day every month, on whichever date your new billing period starts. 
+
+To find the current usage level for any of these metrics, see [retrieving details about a project](#retrieving-details-about-a-project). You can read more about these metrics and how they impact billing [here](/docs/billing).
+
+### Corresponding quotas
+
+You can set quotas for these consumption metrics per project using the following Neon API URL:
+
+```bash
+/projects/{project_id}/settings/quota
+```
+
+The `quota` object includes the array of parameters used to set threshold limits. Their names generally match their corresponding metric:
+
+* `active_time_seconds` &#8212; Sets the maximum amount of wall-clock time allowed in total across all of a project's compute endpoints. This means the total elapsed time, in seconds, from start to finish for each transaction handled by the project's endpoints, accumulated before the one-month refresh date.
+* `compute_time_seconds` &#8212; Sets the maximum amount of CPU seconds allowed in total across all of a project's compute endpoints. This differs from `active_time_seconds` in that it only counts the time the CPU spends executing a process. It excludes time spent waiting for external resources, I/O operations, or time spent in a blocked or idle state.
+* `written_data_bytes` &#8212; Sets the maximum amount of data in total, measured in bytes, that can be written across all of a project's branches for the month.
+* `data_transfer_bytes` &#8212; Sets the maximum amount of egress data, measured in bytes, that can be transferred out of Neon from across all of a project's branches using the proxy.
+
+There is one additional `quota` parameter, `logical_size_bytes`, which applies to individual branches, not to the overall project. You can use `logical_size_bytes` to set the maximum size (measured in bytes) that any one individual branch is allowed to reach. Once this threshold is met, the compute for that particular branch (and _only_ that particular branch) is suspended. Note that this limit is _not_ refreshed once per month: it is a strict size limit that applies for the life of the branch.
+
+### Sample quotas
+
+Let's say you want to set limits for an application with two tiers, Trial and Pro, you might set limits like the following:
+
+| Parameter (project)                 | Trial                     | Pro                      |
+|------------------------|---------------------------|--------------------------|
+| `active_time_seconds`  | 86,400 (1 day)            | 2,592,000 (30 days)      |
+| `compute_time_seconds` | 3,600 (1 hour)            | 50,000 (approx. 14 hours)|
```

Comment:
If my assumptions about compute_time_seconds, I wonder if "1 hour" and "14 hours" makes sense. In our Free Tier, we allow a 1/4 vCPU to run 24/4, for example. So, the setting would be 1/4 of the active_time_seconds value.

Reply:
Does this look a bit better?


| Parameter (project)       | Trial  (.25 vCPU)               | Pro  (max 4 vCPU)                |
|---------------------------|---------------------------------|----------------------------------|
| `active_time_seconds`     | 633,600 (business month 22 days)| 2,592,000 (30 days)              |
| `compute_time_seconds`    | 158,400 (approx 44 hours)       | 10,368,000 (up to 4 times the active time)|
| `written_data_bytes`      | 1,000,000,000 (approx. 1 GiB)   | 50,000,000,000 (approx. 50 GiB)  |
| `data_transfer_bytes`     | 500,000,000 (approx. 500 MiB)   | 10,000,000,000 (approx. 10 GiB)  |

| Parameter (branch)        | Trial                           | Pro                              |
|---------------------------|---------------------------------|----------------------------------|
| `logical_size_bytes`      | 100,000,000 (approx. 100 MiB)   | 10,000,000,000 (approx. 10 GiB)  |